app/models/article.rb

```diff
@@ -0,0 +1,29 @@
+class Article < ApplicationRecord
+  before_validation :generate_handle
+
+  belongs_to :user
```

Comment:
This should either be a `belongs_to: author` (with a foreign-named key of `user_id`).

Or, it should not be included at all and instead, multiple authors should be allowed on an article.  This would mean a `has_many: authors`, and either 1) the `User` model would need to have ids on an `Article`, or 2) a new join model `ArticleAuhor` would need to be added that joins `Article` with an `Author` (along with associations on `Article` and `Author` to get their associated content through `ArticleAuthor`).

Reply:
I really prefer the second option.  The migration converting `belongs_to :user` is a little more convoluted than adding  the second solution later on.

For now, I'm going to _remove_ the author/user association from the `Article` model, so that the second option solution can be added posthumously with greater ease.

Reply:
K, associations from the `Author` have been removed [here](https://github.com/allegroplanet/allegro-planet/pull/64/commits/3d94c2cbb908d80adf576c19f4c3b07d6472378f), and [here](https://github.com/allegroplanet/allegro-planet/pull/64/commits/2adde361fb5b69edb68b557e0f6629aa10df6438).